plugins/genesys/README.md

```diff
@@ -24,7 +24,9 @@ This plugin supports the following protocol IDs:
 
 These devices use the standard USB DeviceInstanceId values for the USB Hub, e.g.
 
-* HP Mxfd FHD Monitor: `USB\VID_03F0&PID_0610`
+* HP USB-C Controller: `USB\VID_03F0&PID_0610`
```

Comment:
The instance ID is not for `HP USB-C Controller`, It is generic with genesys hub.